src/main/python/ttconv/model.py

```diff
@@ -300,16 +300,16 @@ def set_id(self, element_id: str):
   
   # style properties
 
-  def has_style(self, style_prop: StyleProperty) -> bool:
+  def has_style(self, style_prop: typing.Type[StyleProperty]) -> bool:
```

Comment:
Why is this needed? Just curious.

Reply:
Because the `style_prop` passed here is a StyleProperty type, not an instance of it:
We call
```python
element.has_style(Color)
```
instead of
```python
element.has_style(Color())
```